SPECS/libmodulemd/libmodulemd.spec

```diff
@@ -42,11 +40,6 @@ It contains the libraries and header files.
 
 %build
 %meson \
-    -Ddeveloper_build=false \
```

Comment:
Do we know why these should be removed?  Help me understand?  One reason I am asking is because I think these were recently copied to 1.0 too.

Reply:
These arguments are no longer recognized by meson after the upgrade. I spoke to Suresh about them as he added similar arguments for 1.0. He mentioned he doesn't think they should impact functionality at all but recommends validating the changes manually. I will do that as soon as I can but thought I'd give all the details in case you wanted to merge before the build kicks off.